designer/server/src/main/scala/pl/touk/nussknacker/ui/api/description/MigrationApiEndpoints.scala

```diff
@@ -98,6 +115,88 @@ class MigrationApiEndpoints(auth: EndpointInput[AuthCredentials]) extends BaseEn
 object MigrationApiEndpoints {
 
   object Dtos {
+    sealed trait MigrationError
```

Comment:
maybe it'd be better to move it to the `MigrationService` companion object? According to the rule which says the infrastructure layer should have no access to the core/domain layer

Reply:
I tried to follow analogous tapir services. I'll leave TODO for someone to think about it :)

Reply:
please take a look at my improvement suggestions: 
https://github.com/TouK/nussknacker/pull/5975